packages/core/src/input/pointer/Pointer.ts

```diff
@@ -10,13 +12,90 @@ export class Pointer {
    * @remark Start from 0.
    */
   readonly id: number;
+  /** @internal */
+  _events: PointerEvent[] = [];
+
   /** The phase of pointer. */
   phase: PointerPhase = PointerPhase.Leave;
+  /** The button of pointer. */
+  button: PointerButton;
+  /** The buttons of pointer. */
+  buttons: number;
```

Comment:
The API you add `buttons` and `button` comments is not clear, developer can't understand what they are express!

Reply:
These two APIs are required. Taking the mouse as an example, the button represents which key (`Primary`, `Auxiliary`, or `Secondary`) triggers the event, and the buttons represent the `hold down key` of the cursor.